common/src/main/java/io/homeassistant/companion/android/common/data/prefs/PrefsRepositoryImpl.kt

```diff
@@ -20,6 +20,10 @@ class PrefsRepositoryImpl @Inject constructor(
         private const val PREF_LANG = "lang"
         private const val PREF_LOCALES = "locales"
         private const val PREF_SCREEN_ORIENTATION = "screen_orientation"
+        private const val PREF_HEADER_NAME_1 = "header_name_1"
+        private const val PREF_HEADER_NAME_2 = "header_name_2"
+        private const val PREF_HEADER_VALUE_1 = "header_value_1"
+        private const val PREF_HEADER_VALUE_2 = "header_value_2"
```

Comment:
These are server-specific settings as headers should not be shared between servers, and should thus not be added to the `PrefsRepository` (app-level settings) but to the `IntegrationRepository`.

For an example on how to implement this, see the setting 'Remotely control app & device' (`trust_server` in code).